Xamarin.Forms.Controls/GalleryPages/CollectionViewGalleries/ObservableMultiItemCollectionViewGallery.cs

```diff
@@ -0,0 +1,96 @@
+﻿namespace Xamarin.Forms.Controls.GalleryPages.CollectionViewGalleries
+{
+	internal class ObservableMultiItemCollectionViewGallery : ContentPage
+	{
+		public ObservableMultiItemCollectionViewGallery(ItemsLayoutOrientation orientation = ItemsLayoutOrientation.Vertical,
+			bool grid = true, int initialItems = 1000, bool withIndex = false)
+		{
+			var layout = new Grid
+			{ 
+				RowDefinitions = new RowDefinitionCollection
+				{
+					new RowDefinition { Height = GridLength.Auto },
+					new RowDefinition { Height = GridLength.Auto },
+					new RowDefinition { Height = GridLength.Auto },
+					new RowDefinition { Height = GridLength.Auto },
+					new RowDefinition { Height = GridLength.Auto },
+					new RowDefinition { Height = GridLength.Star }
+				}
+			};
+
+			var itemsLayout = grid 
+				? new GridItemsLayout(3, orientation) 
+				: new ListItemsLayout(orientation) as IItemsLayout;
+
+			var itemTemplate = ExampleTemplates.PhotoTemplate();
+
+			var collectionView = new CollectionView {ItemsLayout = itemsLayout, ItemTemplate = itemTemplate};
+
+			var generator = new ItemsSourceGenerator(collectionView, initialItems);
+			
+			var remover = new MultiItemRemover(collectionView, withIndex);
+
+			var adder = new MultiItemAdder(collectionView, withIndex);
```

Comment:
In the test, there is not enough description that the indexes are separated by commas. And it is desirable instead of "0" to fill in the text with template working indexes.
![image](https://user-images.githubusercontent.com/27482193/50957997-b2618780-14d0-11e9-882d-51c9a6433378.png)

Reply:
Updated this to be move obvious (and to include default values).